services/gateway/sources/mappings/account.js

```diff
@@ -27,6 +28,15 @@ module.exports = {
 		rewards: '=,string',
 		username: '=,string',
 		vote: '=,string',
+		totalVotesReceived: ',string',
```

Comment:
`SDKv4 Impact Analysis on Lisk Service` document does not reflect this to be a required change. Request @MichalTuleja to confirm.

Reply:
@MichalTuleja what do you recommend?

Reply:
Given property removed for now based on `SDKv4 Impact Analysis on Lisk Service`